### ISSUE TITLE
[Request]

### ISSUE THREAD

**Issue by JamesDAdams** [Tzahi12345/YoutubeDL-Material#311]
Hi,

I use file output arguments to download all the playlist of a channel which gives that : 

```
.Chaine Name
     .Playlist number 1
          .Video 1
          .Video 2
     .Playlist number 2
          .Video 1
          .Video 2
```
My File output parameters : `%(playlist)s [%(playlist_id)s]/%(playlist_index)s - %(title)s [%(id)s]`
Everything works except viewing the playlist and video on the web interface, it would be nice to add an option to be able to see them :)

![image](https://user-images.githubusercontent.com/20860324/104105174-b89bc180-52ac-11eb-908f-d0ef9f4682a2.png)

Thanks.

**Comment by Tzahi12345** [comment]
Do you mind sharing the URL you used?

**Comment by JamesDAdams** [comment]
> Do you mind sharing the URL you used?

https://www.youtube.com/channel/UCo6bs4WZehBPHfR3bRYuUaQ/playlists

**Comment by Tzahi12345** [comment]
So it seems to download, it's just not separated into distinct playlists. You can do this on your own by grabbing the playlist URL (ex. https://www.youtube.com/playlist?list=PLOyMDfua0nABJtaiv-wviXNm2DVaXo-rQ) and subscribing to each one individually. But, it's a reasonable feature request so if this seems too inconvenient (it has its downsides), I can explore doing this in a future update.

**Comment by JamesDAdams** [comment]
Yes, the download works and the file output options that I put work to download each playlist in a separate folder.
I do this, because then I use plex which recognizes each playlist as a series.

and the whole process is automatic the advantage of placing the url of all the playlists of my youtube channel allows that when I create or subscribe to new playlists they are automatically downloaded with youtubedl without having to add them.
